solutions/kubernetes/cluster-setup.sh

```diff
@@ -63,13 +63,6 @@ function setup_cluster() {
         # Join worker nodes.
         JOIN_WORKER="/var/tmp/cluster-functions.sh --join-worker-nodes $JOIN_COMMAND"
         pdsh -w ^/var/tmp/pdsh-hosts "$JOIN_WORKER"
-
-        # Label worker nodes.
```

Comment:
Is there any specific reason to remove `label` logic?

Reply:
yes, because the `label` logic is failing on worker nodes so will take as improvement